sonarts-core/tests/rules/usePrimitiveReturnTypeRule/usePrimitiveReturnTypeRule.lint.ts

```diff
@@ -0,0 +1,37 @@
+// OK, doesn't have `any`
+function foo() {
+  return 1;
+}
+
+function bar(): any {
```

Comment:
@stas-vilchik I think it will be nice to have more noncomliant examples 
* with other primitive type
* with more than one return
* with return type being inferred from expression (not just literal returned)